.changeset/wise-feet-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": minor
+---
+
+enable margin bottom to the title and change its font size
```

Comment:
I think we just want to mention that we update the fontSize, right?

Reply:
yes, forgot about this, good catch

Reply:
done